app/routes/git_tokens.py

```
@@ -4,340 +4,354 @@
+		encrypted_token = EncryptionHelper.encrypt(token) if token else ""
+		
+		if payload.git_hosting == GitHosting.GITLAB.value:
+			return await handle_gitlab(payload, encrypted_token)
+		elif payload.git_hosting == GitHosting.GITHUB.value:
+			return await handle_github(payload, encrypted_token)
+		else:
+			return APIResponse.error(
+				message=constants.UNSUPPORTED_GIT_PROVIDER,
+				status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+			)
+	except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)